SpatialGDK/SpatialGDK.uplugin

```diff
@@ -72,6 +72,15 @@
       "WhitelistPlatforms": [
         "Win64"
       ]
+    },
+    {
+      "Name": "SpatialGDKFunctionalTests",
+      "Type": "Editor",
+      "LoadingPhase": "Default",
+      "WhitelistPlatforms": [ 
+        "Win64",
+	"Mac"
```

Comment:
Weird whitespace in GH here.

Reply:
Just highlighting Win is always ahead of Mac... will fix!